packages/docusaurus-theme-classic/src/theme/IconClose/index.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import type {Props} from '@theme/IconMenu';
+
+export default function IconClose({
+  width = 20,
+  height = 20,
+  className,
+  ...restProps
+}: Props): JSX.Element {
+  return (
+    <svg
```

Comment:
As mentioned above, this icon is more suitable in combination with menu icon:

```html
<svg viewBox="0 0 413.348 413.348" width="20" height="20">
  <path d="m413.348 24.354-24.354-24.354-182.32 182.32-182.32-182.32-24.354 24.354 182.32 182.32-182.32 182.32 24.354 24.354 182.32-182.32 182.32 182.32 24.354-24.354-182.32-182.32z"/>
</svg>
```

Reply:
So we'd use 2 distinct icons? Apart from the duplicated, I tested this and it looks weird to have different icons, particularly because both can be displayed at the exact same time (the announcement bar close icon can be visible through the navbar-sidebar overlay).

I'd prefer to keep the exact same one in both places for consistency and simplicity (but I don't mind using one or another)

Reply:
Ideally, it is better to use different "x" icons, because for mobile navbar we need icon with thick lines, and for mobile navbar we need icon with thin lines.

Reply:
> because for mobile navbar we need icon with thick lines, and for mobile navbar we need icon with thin lines.

I'll add back the 2 distinct svgs but not sure it's much better

Reply:
Thanks, I like to have uniformity in design, that's why icons in navbar area should be similar in style, isn't it?  In the future, it's probably better to have single one SVG icon, in which it would be possible to set `stroke-width`.